Assets/Scripts/Robo.cs

```diff
@@ -31,7 +41,7 @@ void Start()
         _sprite = GetComponent<SpriteRenderer>();
         _rigidbody = GetComponent<Rigidbody2D>();
 
-
+        curr_health = max_health;
```

Comment:
Same as other comments, should we initalize this at declaration?

Reply:
This warrants its own reply comment from me because it's different from previous instances of curr_health initialization.

I could actually do that if I make max_health a static variable. The main reason to not do so is that max_health  may change in runtime with future features (for example, armor or another progression system increases a player's max health). However, it could be done.

Reply:
Oh shoot, you right. I got it slightly mixed up with const.

I don't think there are any downsides to doing this that I can think of, and you seem not to have any, so I'll make it a static variable and push the change.